modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/field-types/FieldType.es.js

```diff
@@ -122,16 +122,13 @@ export default (props) => {
 			</ClayLayout.ContentCol>
 
 			<ClayLayout.ContentCol className="pr-2" expand>
-				<div className="list-group-title text-truncate">
-					<span>
-						{label}
-
-						{required && (
-							<span className="reference-mark">
-								<ClayIcon symbol="asterisk" />
-							</span>
-						)}
-					</span>
+				<div className="d-flex list-group-title">
+					<span>{label}</span>
```

Comment:
@MateusCastro did try to add the 'text-truncate' classname instead of doing it in the css?

Reply:
It really gets better. I will update the code. 

Thanks @interaminense